src/main/java/nl/workingtalent/wtacademy/author/Author.java

```diff
@@ -20,7 +22,8 @@ public class Author {
 	@Column(length = 100)
 	private String name;
 	
-	@ManyToMany
+	@JsonIgnore
+	@ManyToMany(mappedBy = "authors")
 	private List<Book> books;
```

Comment:
Waarom @JSONignore ipv Dto?

Reply:
Dit komt van main vandaan. Is nog de oude code die geschreven was voordat wij Martijn gesproken hadden